validator/db/kv/attestation_history_v2.go

```diff
@@ -59,19 +61,42 @@ func (store *Store) AttestationHistoryForPubKeyV2(ctx context.Context, publicKey
 }
 
 // SaveAttestationHistoryForPubKeyV2 saves the attestation history for the requested validator public key.
-func (store *Store) SaveAttestationHistoryForPubKeyV2(ctx context.Context, pubKey [48]byte, history EncHistoryData) error {
+func (store *Store) SaveAttestationHistoryForPubKeyV2(
+	ctx context.Context,
+	pubKey [48]byte,
+	history EncHistoryData,
+	lowestSourceEpoch,
+	lowestTargetEpoch uint64,
+) error {
 	ctx, span := trace.StartSpan(ctx, "Validator.SaveAttestationHistoryForPubKeyV2")
 	defer span.End()
-	err := store.update(func(tx *bolt.Tx) error {
-		bucket := tx.Bucket(newHistoricAttestationsBucket)
-		return bucket.Put(pubKey[:], history)
-	})
+	go func() {
```

Comment:
this is an experiment. seems to work better but still if io cant handle the load it just postpones the issue